base/range.jl

```diff
@@ -182,33 +183,33 @@ end
 
 function linspace{T<:AbstractFloat}(start::T, stop::T, len::T)
```

Comment:
It would sound more logical to me to allow `len` to be of a different type from `start` and `stop`, than to introduce the notion of "unitful length" (which doesn't mean anything AFAICT :-).